ui/non_interactive.go

```diff
@@ -40,11 +40,11 @@ func (ui *NonInteractiveUI) PrintTable(table Table) {
 	ui.parent.PrintTable(table)
 }
 
-func (ui *NonInteractiveUI) AskForText(label string) (string, error) {
+func (ui *NonInteractiveUI) AskForText(opts TextOpts) (string, error) {
```

Comment:
@cppforlife Since we have started passing `default Value` in the `textOpts`, should this function continue to panic or it should return the default value? What becomes interesting is, that on the terminal we will not show the label and the default value to the user in non-interactive mode.

Reply:
We will never know whether the empty value is intentional as a default value or not. It becomes more interesting in case of `AskForChoice`. DefaultValue can be an empty string which may not be even present in choices.

Reply:
I think in the case of `AskForText`, we return default Value. And in the case of `AskForChoice`, we return if default value is present in the slices of choices, otherwise we panic or return error. Not sure if we should panic or return error as this is library and if it panic, we wont be able to shutdown gracefully.